test-framework/junit5/src/main/java/io/quarkus/test/junit/QuarkusMock.java

```diff
@@ -56,4 +57,16 @@ public static <T> void installMockForType(T mock, Class<? super T> instance, Ann
         }
         MockSupport.installMock(CDI.current().select(instance, qualifiers).get(), mock);
     }
+
+    /**
+     * Installs a mock for a CDI normal scoped bean by typeLiteral and qualifiers
+     *
+     * @param mock          The mock object
+     * @param typeLiteral   TypeLiteral representing the required type qualifiers – the additional required qualifiers
```

Comment:
Pls remove the `qualifiers – the additional required qualifiers` part. The type literal has nothing to do with qualifiers ;-).

Reply:
@mkouba done

Reply:
You misunderstood me - we should keep the `@param qualifiers` but remove the `qualifiers – the additional required qualifiers` part from the `@param typeLiteral`  line ;-)

Reply:
@mkouba done again :)